source/reference/supported-server-version.txt

```diff
@@ -0,0 +1,83 @@
+.. _c2c-server-version-compatibility:
+
+================================================
+MongoDB Server Version Compatibility and Support
+================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Before you run {+c2c-product-name+}, consider the following MongoDB server 
+version limitations and requirements: 
+
+- .. include:: /includes/fact-minimum-server-version-support.rst
+
+- ``mongosync`` doesn't support MongoDB rapid releases, such as 6.1 or 6.2.
+
+- .. include:: /includes/fact-minimum-fcv.rst 
+
+Synchronize Data Between Clusters Running Older MongoDB Server Versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/fact-minimum-versions.rst
+
+
+Synchronize Data Between Clusters with Different MongoDB Server Major Versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/cross-version-sync.rst
+
+``mongosync`` supports syncs between the following source and
+destination MongoDB server versions. The table shows the source MongoDB
+server versions in the left column and destination MongoDB server
+versions in the first row. 
+
+.. list-table:: 
+   :widths: 20 20 20 20 20
+
+   * - 
+     - **4.2**
+     - **4.4**
+     - **5.0**
+     - **6.0**
+     - **7.0**
+
+   * - **4.2**
```

Comment:
I think we should check with PM on including 4.2 in this table, since it went EOL last year.

If we're not removing mention altogether, we may want to remove it as an option for destination clusters, since that's giving the reader the idea that they can sync their data onto an unsupported release.